### ISSUE TITLE
Components as Icons

### ISSUE THREAD

**Issue by eniacada** [TatuLund/tree-flow#10]
![image](https://github.com/user-attachments/assets/f22c0eea-55c8-487c-adcd-cea9ef1743de)
I would like to add a progress bar as an icon in the leaflet.  What is the best possible way to achieve this.  Sorry I know this is not exactly an issue but, I did not see another way for to communicate this for this addon.

**Comment by TatuLund** [comment]
No, this is not supported in Tree. I.e. there is no component renderer option.
